tests/AppServiceProvider.txt

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
it nags me that this file is named super generic and has no documentation tied to it other than at line in tests/TestCase.php.